app/app/src/main/java/me/bickositieff/raspio/ui/models/Song.kt

```diff
@@ -0,0 +1,15 @@
+package me.bickositieff.raspio.ui.models
+
+import androidx.annotation.DrawableRes
+
+class Song(
+    val title: String,
+    val artist: String,
+    val duration: Int,
+    @DrawableRes
+    val image: Int?
```

Comment:
Will probably not be usable like that but we'll change it when we know how we'll use it